test/spinxsk/spinxsk.c

```diff
@@ -2307,19 +2306,19 @@ InjectFnmpRxFrames(
         fnmpFrame.Input.Checksum.Value = (PVOID)(ULONG_PTR)RandUlong();
         fnmpFrame.Input.Rsc.Value = (PVOID)(ULONG_PTR)RandUlong();
         fnmpFrame.Buffers = buffers;
-        fnmpFrame.BufferCount = numBuffers + 1;
+        fnmpFrame.BufferCount = numBuffers;
 
-        FnMpRxEnqueue(Fnmp, &fnmpFrame);
+        status = FnMpRxEnqueue(Fnmp, &fnmpFrame);
+        TraceVerbose("FnmpInjectRx: FnMpRxEnqueue failed with %d", status);
```

Comment:
Nit: we're super inconsistent, but I generally try to print HRESULTs as `%x` or `%!HRESULT!`